bin/horton-convert

```diff
@@ -36,7 +36,7 @@ def parse_args():
                                                  ' This only works if the input contains sufficient'
                                                  ' data for the output')
     parser.add_argument('-V', '--version', action='version',
-                        version="%%(prog)s (HORTON version %s)" % version.__version__)
+                        version="%%(prog)s (HORTON version 2.0)" )
```

Comment:
We used this in the past so you could define version numbers using git tags. It was too easy to neglect to update a line somewhere and have inconsistent version numbers in the program. 

The downside is that if you didn't use the travisCI infrastructure, the version.py file didn't get generated. It's probably better to use a `try: ... except ImportError ...` fallback to get around this instead of hard coding the number.

Reply:
Yeah, we should make this easier. I'm working on something (roberto), but is not quite ready yet.

Reply:
I did add a try: ... except ImporError as suggested by Matt. Hope this is fine. Thanks @matt-chan